.github/workflows/cli-scripts-batch-score.yml

```diff
@@ -25,8 +25,6 @@ jobs:
       run: bash setup.sh
```

Comment:
I would suggest only keep change of the spark ones. let others to upgrade their workflow

Reply:
removed